public_html/wp-content/plugins/pattern-directory/includes/pattern-taxonomy-modifications.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace WordPressdotorg\Pattern_Directory\Pattern_Taxonomy_Modifications;
+
+$category_taxonomy = 'wporg-pattern-category';
+const CATEGORY_TAXONOMY_ID = 'display-order';
+
+add_action( "{$category_taxonomy}_add_form_fields", __NAMESPACE__ . '\add_ordering_form_field', 0 );
+add_action( "{$category_taxonomy}_edit_form_fields", __NAMESPACE__ . '\edit_ordering_form_field', 10, 2 );
+add_action( "created_{$category_taxonomy}", __NAMESPACE__ . '\save_term_field' );
+add_action( "edited_{$category_taxonomy}", __NAMESPACE__ . '\save_term_field' );
+add_filter( "manage_{$category_taxonomy}_custom_column", __NAMESPACE__ . '\add_category_order_column_content', 10, 3 );
+add_filter( "manage_edit-{$category_taxonomy}_columns", __NAMESPACE__ . '\add_order_table_heading' );
+add_filter( "rest_prepare_{$category_taxonomy}", __NAMESPACE__ . '\rest_prepare_category_response', 10, 2 );
+
+/**
+ * Adds a form field for adding an ordinal number to a new category.
+ *
+ * This is a callback for the `{$taxonomy}_add_form_fields` filter, and it's used to modify the new taxonomy form.
+ */
+function add_ordering_form_field() {
+	echo '
+		<div class="form-field">
+			<label for="display-order">Display Order</label>
+			<input type="number" min="1" name="display-order" id="display-order" />
+			<p>This is used to order the categories in the pattern grid navigation.</p>
+		</div>
+	';
+}
+
+/**
+ * Adds a form field for editing an ordinal number for a category.
+ *
+ * This is a callback for the `{$taxonomy}_edit_form_fields` filter, and it's used to modify the edit taxonomy form.
+ *
+ * @param WP_Term $term Term object.
+ */
+function edit_ordering_form_field( $term ) {
+	$value = get_term_meta( $term->term_id, CATEGORY_TAXONOMY_ID, true );
+
+	echo '
+		<tr class="form-field">
+			<th>
+				<label for="display-order">Display Order</label>
+			</th>
+			<td>
+				<input name="display-order" min="1" id="display-order" type="number" value="' . esc_attr( $value ) . '" />
+				<p class="description">This is used to order the categories in the pattern grid navigation.</p>
+			</td>
+		</tr>
+	';
+
+}
+
+/**
+ * Saves the custom taxonomy field to the database.
+ *
+ * This is a callback for the `created_{$taxonomy}` and `edited_{$taxonomy}` filter, and it's used to save meta information to the database.
+ *
+ * @param integer $term_id Term id.
+ */
+function save_term_field( $term_id ) {
+	/* phpcs:disable WordPress.Security.NonceVerification.Missing */
```

Comment:
I had to disable this rule. 

@iandunn Is it safe to assume we don't need to verify the nonce here?

Reply:
I think [the best practice is to use a nonce](https://wordpress.stackexchange.com/questions/49453/do-i-need-a-nonce-field-for-every-meta-box-i-add-to-my-custom-post-type-admin). It won't be necessary in all cases, but can be sometimes. It also makes security review easier, similar to late escaping.

Similarly, it should probably also use `current_user_can()` to make sure the user submitting the request is allowed to edit terms for this taxonomy.